ballerina/websocket_service.bal

```diff
@@ -81,34 +79,37 @@ isolated service class WsService {
                 }
             }
             WS_SUBSCRIBE|WS_START => {
+                if wsPayload.id is () {
+                    return self.handleIdNotPresentInPayload(caller);
+                }
+                string connectionId = <string>wsPayload.id;
```

Comment:
We should avoid castings like this. It is considered bad practice in Ballerina. The better way to handle this is this:

```suggestion
                string? connectionId = wsPayload.id;
                if connectionId is () {
                    return self.handleIdNotPresentInPayload(caller);
                }
                // ...
```

Assigning `wsPayload.id` to a local variable will nil-lift after returning.

Reply:
updated